docs/start/framework/solid/build-from-scratch.md

```diff
@@ -5,5 +5,10 @@ replace:
     '@tanstack/react-start': '@tanstack/solid-start',
     'React': 'SolidJS',
     'react-router': 'solid-router',
+    'react react-dom': 'solid-js',
```

Comment:
so i guess it just string replaces the docs? I wonder how we would add the viteSolid({ssr: true}) part

Reply:
I think there are some bigger differences like not hydrating the whole html document